easybuild/easyconfigs/r/R/R-4.0.0-foss-2020a.eb

```diff
@@ -2247,8 +2245,7 @@ exts_list = [
         'checksums': ['4152a1c3c652979e97870e5c50c45a243d0ad8d4ff968091160e3d66509f61db'],
     }),
     ('rda', '1.0.2-2.1', {
-        'checksums': [('6918b62f51252b57f2c05b99debef6136b370f594dc3ae6466268e4c35578ef8',
-                       'eea3a51a2e132a023146bfbc0c384f5373eb3ea2b61743d7658be86a5b04949e')],
```

Comment:
Please keep both (alternate) checksums, this is the fault of `--inject-checksums`...

Reply:
Fixed.